tests/unit/test_backups.py

```diff
@@ -1813,7 +1813,8 @@ def test_render_pgbackrest_conf_file(harness, tls_ca_chain_filename):
             storage_path=harness.charm._storage_path,
             user="backup",
             retention_full=30,
-            process_max=max(cpu_count() - 2, 1),
+            process_max=2,
```

Comment:
why hardcoded 2?

Reply:
We should be getting the number of available cores from `get_available_resources()` in case of fractional limits and the output of it is patched above.